jenkins/clean_project.py

```diff
@@ -57,8 +57,8 @@ def DeleteInstances(project, zones, delete):
   return err
 
 
-def main(project, days, filt, delete):
-  age = datetime.datetime.now() - datetime.timedelta(days=int(days))
+def main(project, hours, filt, delete):
+  age = datetime.datetime.now() - datetime.timedelta(hours=int(hours))
```

Comment:
True... I think this was an artifact from before I started using argparse and was just doing *sys.argv[1:] :dancers: